base/src/main/java/uk/ac/ebi/atlas/utils/ResourceUtils.java

```diff
@@ -5,11 +5,12 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
 
 public class ResourceUtils {
```

Comment:
is the "Resouce" the Spring resource only, like, "avgExpressionOverBioentityIdentifiers.json" ? It seems a bit accidental why UTF_8 / where this can be used but if it works it works.